packages/images-plugin/lib/finder.js

```diff
@@ -103,34 +107,30 @@ export default class ImageElementFinder {
    */
   registerImageElement(imgEl) {
     this.imgElCache.set(imgEl.id, imgEl);
-    const protoKey = this.driver.isW3CProtocol() ? W3C_ELEMENT_KEY : MJSONWP_ELEMENT_KEY;
-    return imgEl.asElement(protoKey);
+    if (!this.driver) {
+      throw new ReferenceError('No driver set!');
```

Comment:
would it be possible to provide more helpful error description?

Reply:
also I personally don't like to put exclamation marks into error messages: too much drama %)

Reply:
yeah, I blame copilot.  I think saying something like "call setDriver() first` would work